src/common/utils.js

```diff
@@ -100,10 +103,32 @@ export function copyToClipboard(data) {
   textField.remove();
 };
 
+/* Returns date set to local timezone and in format of locale */
```

Comment:
can you add unit tests for these functions

Reply:
added unit tests